src/tests/system/tests/test_authentication.py

```diff
@@ -15,55 +15,60 @@
     :setup:
-        1. Add user to SSSD
-        2. Set password for user
-        3. Start SSSD
+        1. Create user
+        2. Start SSSD
     :steps:
         1. Authenticate user with correct password
         2. Authenticate user with incorrect password
     :expectedresults:
-        1. User is authenticated
-        2. User is not authenticated
+        1. Authentication is successful
+        2. Authentication is unsuccessful
     :customerscenario: False
     """
     provider.user("user1").add(password="Secret123")
 
     client.sssd.start()
 
-    assert client.auth.parametrize(method).password("user1", "Secret123"), "login with correct password failed"
-    assert not client.auth.parametrize(method).password("user1", "NOTSecret123"), "login with wrong password succeeded"
+    assert client.auth.parametrize(method).password("user1", "Secret123"), "User failed login!"
+    assert not client.auth.parametrize(method).password(
+        "user1", "NOTSecret123"
+    ), "User logged in with an invalid password!"
 
 
 @pytest.mark.topology(KnownTopologyGroup.AnyProvider)
 @pytest.mark.parametrize("method", ["su", "ssh"])
-def test_authentication__using_a_good_then_bad_password_when_offline(
-    client: Client, provider: GenericProvider, method: str
+@pytest.mark.importance("critical")
+def test_authentication__default_settings_when_the_provider_is_offline(
+    client: Client,
+    provider: GenericProvider,
+    method: str,
 ):
     """
-    :title: Offline ssh/su login
+    :title: Authenticate with default settings when the provider is offline
     :setup:
-        1. Add user to SSSD and set its password
-        2. In SSSD domain change "cache_credentials" and "krb5_store_password_if_offline" to "True"
-        3. In SSSD pam change "offline_credentials_expiration" to "0"
-        4. Start SSSD
+        1. Create user
+        2. Configure SSSD with "cache_credentials = true" and "krb5_store_password_if_offline = true" and
+        "offline_credentials_expiration = 0"
+        3. Start SSSD
     :steps:
-        1. Authenticate user with wrong password
-        2. Authenticate user with correct password
-        3. Make server offline (by blocking traffic to the provider)
-        4. Bring SSSD offline explicitly
-        5. Offline authentication of user with correct password
-        6. Offline authentication of user with wrong password
+        1. Authenticate user with correct password
+        2. Block outbound traffic to the provider and force SSSD offline
+        3. Authenticate user with correct password
+        4. Authenticate user with incorrect password
     :expectedresults:
-        1. User is not authenticated
-        2. User is authenticated
-        3. Firewall rule added, traffic is dropped.
-        4. SSSD is offline
-        5. Offline authentication is successful
-        6. Offline authentication is not successful
+        1. User authentication is successful
```

Comment:
This do not feel right. I would say that this describes the steps just with another words. Particularly  "2. No traffic is getting to the provider" - we just set this, we do not test that with tcpdump or so. What about

1. Online authentication with correct password is succesfull
2. Offline authentication with correct password is succesfull
3. Offline authentication with incorrect password fails

Reply:
I like it, I don't know how other people will feel about it, because any step that needs to be performed has been provided historically in the steps. Then the arbitrary result is necessary because Polarion requires the same number of expected results. 

For this PR, let's leave since it's a cherry pick. Here is a PR fixing the language for master and backports. https://github.com/SSSD/sssd/pull/7533

Reply:
I forgot that this is cherry-picking, sorry. Then I prefer to keep it the same as the source.